quickjs.c

```diff
@@ -40573,18 +40581,56 @@ static JSValue js_iterator_helper_next(JSContext *ctx, JSValue this_val,
             goto done;
         }
         break;
+    case JS_ITERATOR_HELPER_KIND_FILTER:
+        {
+            JSValue item, method, selected, index_val, args[2];
+            if (magic == GEN_MAGIC_NEXT) {
+                method = js_dup(it->next);
+            } else {
+                method = JS_GetProperty(ctx, it->obj, JS_ATOM_return);
+                if (JS_IsException(method))
+                    goto fail;
+            }
+            filter_again:
```

Comment:
For consistency with existing code:
```suggestion
        filter_again:
```

Reply:
I'll eventually get it right lol